### PR TITLE
Launch smoke tests periodacally from container + update gitignore

### PR DESCRIPTION
**Pull request by jhadvig**
Rewrote server.js so it would re-run tests periodically, every 5 minutes. This interval can be  overridden  with `TEST_INTERVAL`, which represents number of minutest that shall take place between each test.

Removed artifacts from `test_results` folder + Updated README + updated gitignore so the `test_results` folder will be ignored.

@benjaminapetersen PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ CONSOLE_URL=https://<machine-ip>:8443 \
 ```bash
 $ ./docker_build.sh
 # or
-$ docker build -t protractor-smoke-test .
+$ docker build -t origin-web-console-smoke-test .
 ```
 
 # Run the tests in a container
 
 ```bash
```
Comment (benjaminapetersen) [inline README.md]: Shall we add `TEST_INTERVAL` to README?  Assuming its in `minutes`, unless we wanted to support something like `TEST_INTERVAL=2m` or `TEST_INTERVAL=1h`, but I dont think thats likely necessary.

File: test/server.js [hunk 2]
```diff
@@ -52,29 +55,25 @@ var fileReadInterval = setInterval(() => {
...
-  console.log(`GET /metrics ${register.metrics()}`);
-  res.set('Content-Type', register.contentType);
-  res.end(register.metrics());
-});
-
-server.get('/protractor', (req, res) => {
-  var sys = require('util');
-  var exec = require('child_process').exec;
+var smokeTestsInterval = setInterval(() => {
+  console.log("[INFO] Running protractor tests in interval");
```
Comment (benjaminapetersen) [inline test/server.js]: Shall we log the interval here:

```
console.log(`[INFO] Running smoke tests at interval (${TEST_INTERVAL})`);
```

File: test/server.js [hunk 3]
```diff
@@ -52,29 +55,25 @@ var fileReadInterval = setInterval(() => {
...
+      // clearInterval(smokeTestsInterval);
     }
     console.log('[INFO] ' + stdout);
   });
-  res.end();
+}, env.test_interval);
+
+
+// prometheus should scrape this endpoint for data.
+server.get('/metrics', (req, res) => {
```
Comment (benjaminapetersen) [inline test/server.js]: Wondering now, per @spadgett's comment, if the `/metrics` enpoint also needs a rate limit to ensure no DOS attack.

Comment (jhadvig) [reply test/server.js]: @benjaminapetersen definitely would be god to have the limit there as well.

**Pull request merged**